CMakeLists.txt

```diff
@@ -1,14 +1,16 @@
-cmake_minimum_required(VERSION 3.22)
-set(CMAKE_CXX_STANDARD 17)
+cmake_minimum_required(VERSION 3.15)
 
 project(Ship)
 
-file(GLOB_RECURSE SHIP_SOURCES src/*.[hc]pp)
-add_executable(Ship ${SHIP_SOURCES})
+file(GLOB_RECURSE SHIP_SOURCES src/*.cpp)
```

Comment:
?

Reply:
The file currently supports older versions of CMake (Testend on 3.15.7)

Reply:
this change loses .hpp files, if you want to support older cmake then make a replacement

Reply:
Changed